### ISSUE TITLE
Should preferences dialog open with screen reader?

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/john-travoltage#442]
**Test device**
Dell
**Operating System**
Win 10
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/qa/issues/676. Seen with Jaws.
When using Chrome + Jaws + keyboard nav, I was unable to open the preferences dialog. I was able to open it with Firefox + NVDA and iPad +VO, so I'm not sure what the desired behavior is.

**Visuals**

https://user-images.githubusercontent.com/41024075/127717258-b792923f-35c4-4752-83e0-be2fc80a8350.MOV

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪John Travoltage‬
URL: https://phet-dev.colorado.edu/html/john-travoltage/1.6.0-rc.1/phet/john-travoltage_all_phet.html
Version: 1.6.0-rc.1 2021-07-22 18:05:38 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.107 Safari/537.36
Language: en-US
Window: 1280x657
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by jessegreenberg** [comment]
Thanks @KatieWoe, yes it should. Ill investigate.

**Comment by jessegreenberg** [comment]
Confirmed with JAWS in Chrome. When activating, we get three events
![image](https://user-images.githubusercontent.com/6396244/127887283-2f0767a7-07a0-4964-b47d-6e483da8fa0b.png)

**Comment by jessegreenberg** [comment]
What is happening in this case is that the JAWS event target is not under the PDOM. So this block is preventing scenery from dispatching the up/down pointer events
```js
    if ( this.isTargetUnderPDOM( event.target ) ) {
      return;
    }
```

But no matter if we dispatch, the event we save the enent.timeStamp on the pointerUp, and so we block the click event too with
```js
            if ( trail && !( _.some( trail.nodes, node => node.positionInPDOM ) && eventName === 'click' &&
                 event.timeStamp - this.upTimeStamp <= PDOM_CLICK_DELAY ) ) {
              this[ actionName ].execute( event );
            }
```

**Comment by jessegreenberg** [comment]
If I remove the `isTargetUnderPDOM` checks, it works OK in JAWS, the same workaround with PDOM_CLICK_DELAY works for that platform.

**Comment by jessegreenberg** [comment]
Alright, this needs to move to scenery.

**Comment by KatieWoe** [comment]
This seems fixed in rc.2. However, I wonder if https://github.com/phetsims/john-travoltage/issues/447 is related, since I found it while looking at this bug.

**Comment by jessegreenberg** [comment]
Thanks @KatieWoe, sounds like this was fixed and we determined in #447 that this issue was not related. Closing this one.
